yellowbrick/features/pca.py

```diff
@@ -0,0 +1,103 @@
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+##########################################################################
+##2D PCA Visualizer
+##########################################################################
+class PCA2D(DataVisualizer):
+    """
+    Two dimensional principal component (PC) plot of data projected onto the first and
+    second principal components. It is best practices to center and scale the inputted
+    data set before applying a PC decomposition. There are scale and center arguments
+    that can be used to control centering anc scaling of an inputted data set. Therefore 
+    this class is a one stop shop for easily getting a 2 dimensional PC plot.
+
+    Parameters
+    ----------
+
+    ax : matplotlib Axes, default: None
+        The axes to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+    
+    scale : bool, default: True
+    
+    center : bool, default: True
+    
+    col : list or tuple, default: None
+        optional list or tuple of colors that distinguish separate classes in
+        int the data set.
+
+    colormap : string or cmap, default: None
+        optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+    
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+    >>> from sklearn import datasets
+    >>> iris = datasets.load_iris()
+    >>> X = iris.data
+    >>> y = iris.target
+    >>> params = {'scale': True, 'center': False, 'col': y}
+    >>> visualizer = PCA2D(**params)
+    >>> visualizer.fit(X)
+    >>> visualizer.transform(X)
+    >>> visualizer.poof()
+
+
+    Notes
+    -----
+    
+    
+    """
+    def __init__(self, ax=None, scale=True, center=True, col=None,
+                 colormap='RdBu_r', **kwargs):
+        super(PCA2D, self).__init__(ax=ax, **kwargs)
+        # Data Parameters
+        self.col = col
+        self.pca_features_ = None
+        self.scale = scale
+        self.center = center
+
+        self.pca_transformer = Pipeline([('scale', StandardScaler(with_mean=self.center,
+                                                                  with_std=self.scale)),
+                                         ('pca', PCA(2))])
+        # Visual Parameters
+        self.colormap = colormap
+
+    def fit(self, X, y=None, **kwargs):
+        self.pca_transformer.fit(X)
+        return self
+
+    def transform(self, X, y=None, **kwargs):
+        self.pca_features_ = self.pca_transformer.transform(X)
+        self.draw()
+        return self.pca_features_
+
+    def draw(self, **kwargs):
+        X = self.pca_features_
+        if self.ax is None:
+            self.ax = self.gca()
```

Comment:
Hmm, I went into your branch and it looks like you have the latest code. These two lines of code if they're in should cause an error at this point. Can you describe the error you were seeing?